test/test_cuda.py

```diff
@@ -1070,6 +1070,10 @@ def test_mul(dtype):
         test_mul(torch.float32)
         test_mul(torch.float64)
 
+        a1 = torch.tensor([True, False, False, True], dtype=torch.bool, device='cuda')
```

Comment:
this doesn't seem like the correct place to test this:
1) This is only executed on cuda code
2) This test is for inter-dtype computations, but everything here is a bool.